misc/vim/syntax/nit.vim

```diff
@@ -86,9 +86,9 @@ syn match  NITSharpBang	"\%^#!.*"
 syn match  NITComment	"#.*" contains=NITTodo,@Spell
 
 " Keywords
-syn keyword NITKeyword	 abstract intern new
-syn keyword NITDefine	 private public protected intrude readable writable redef
```

Comment:
OK to remove `readable` but keep `writable`.

Don't worry (or touch) to other annotations. I see what bothers you, I'll look for alternatives and talk with the others Vim users about what behavior they prefer.